index.html

```diff
@@ -43,6 +43,179 @@ <h1 class="text1">Hey There.<br>I'm Chris.</h1>
+                  </p>
+                  <ul class="sec-grid-item-3">
+                    <li><button class="button"><a href="#" class="but">Html</a></button></li>
+                    <li class="but"><button class="button"><a href="#" class="but">bootstrap</a></button></li>
+                    <li class="but"><button class="button"><a href="#" class="but">Ruby</a></button></li>
+                  </ul>
+                  <button class="sec-grid-item-5"><a href="index.html" class="see-project-t">see project</a></button>
+                </section>
+                </div>
+                <div class="sec-grid-container">
+                  <section class="background">
+                  <h2 class="sec-grid-item-1 text-1">
+                    Profesional Art <br> Printing Data
+                  </h2>
+                  <p class="sec-grid-item-2 text-2">
+                    A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required. Has been the <br> industry's standard.
+                  </p>
+                  <ul class="sec-grid-item-3">
+                    <li><button class="button"><a href="#" class="but">Html</a></button></li>
+                    <li class="but"><button class="button"><a href="#" class="but">bootstrap</a></button></li>
+                    <li class="but"><button class="button"><a href="#" class="but">Ruby</a></button></li>
+                  </ul>
+                  <button class="sec-grid-item-5"><a href="index.html" class="see-project-t">see project</a></button>
+                </section>
+                </div>
+                <div class="last-grid-container">
+                  <section class="background">
+                  <h2 class="sec-grid-item-1 text-1">
+                    Profesional Art <br> Printing Data
+                  </h2>
+                  <p class="sec-grid-item-2 text-2">
+                    A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required. Has been the <br> industry's standard.
+                  </p>
+                  <ul class="sec-grid-item-3">
+                    <li><button class="button"><a href="#" class="but">Html</a></button></li>
+                    <li class="but"><button class="button"><a href="#" class="but">bootstrap</a></button></li>
+                    <li class="but"><button class="button"><a href="#" class="but">Ruby</a></button></li>
+                  </ul>
+                  <button class="sec-grid-item-5"><a href="index.html" class="see-project-t">see project</a></button>
+                </section>
+                </div>
+                <div class="about-section">
+                  <section class="about-grid">
+                  <nav class="about-h1-section">
+                    <h1 class="h1-text">
+                      About me
+                    </h1>
+                  </nav>
```

Comment:
- [ ] Please do not wrap the h1 element inside a `nav`. The `nav` should be used only to wrap a list of an element that helps to navigate through the website. Also, consider changing `h1` to `h2` as each page should have only one `h1`. This will improve greatly the semantic of your webpage 😄 which will make your website much more accessible (for users, screen readers, and search engines).